src/block/Bamboo.php

```diff
@@ -124,7 +124,8 @@ public function getModelPositionOffset() : ?Vector3{
 	}
 
 	private function canBeSupportedBy(Block $block) : bool{
-		return $block->hasTypeTag(BlockTypeTags::DIRT) || $block->hasTypeTag(BlockTypeTags::MUD);
+		// TODO: Gravel
```

Comment:
You can just check for the ID. Tags are only used when groups of similar blocks are found.